### DISCUSSION TITLE
change default parameters for the web popup

### DISCUSSION THREAD

**Discussion by aestheticjmack** [Chatterino/chatterino2#2776] [Ideas]
The default query parameters for the player popup doesn't allow extensions, which requires manually changing the url.

The url is 
https://player.twitch.tv/?channel=**%CHANNELNAME%**&enableExtensions=true&muted=true&parent=twitch.tv&player=popout&volume=1

fourtf: edited to reflect the actual issue

**Comment by KararTY** [comment]
Are you looking for this?
![image](https://user-images.githubusercontent.com/13697809/118115487-4ae1dc80-b3e9-11eb-89e9-2505195856da.png)
This will open the stream player in your default browser for the selected split. Right click the split header (or the 3 dots / hamburger menu) to open this menu up.

**Reply by aestheticjmack** [reply]
This is a QOL request really, as an example I want it to open straight to here

![image](https://user-images.githubusercontent.com/70598029/118116768-4fd66a00-b3b0-11eb-8c2c-b6909e70961c.png)

**Comment by fourtf** [comment]
I assume you want these specific parameters in the link? Otherwise I don't see how this is different from the already existing menu option in the dropdown.

**Reply by aestheticjmack** [reply]
Yes, exactly

**Reply by fourtf** [reply]
I edited the issue to reflect this

**Reply by aestheticjmack** [reply]
thanks, your team does great work! (:
